src/App.tsx

```diff
@@ -4,29 +4,28 @@ import logo from "../public/FischLogo.svg";
 
 import styles from "./App.module.css";
 import { Card } from "./components/Card";
-import { useState } from "react";
+import { useRef, useState } from "react";
 
 function App() {
+  const [cardsAmount, setCardsAmount] = useState(1)
   const [cards, setCards] = useState([{}]);
 
   const handleAddCardButtonClick = () => {
+    setCardsAmount(cardsAmount+1)
     setCards(prevCards => [...prevCards, {}]);
   };
-
-  const handleDeleteCardButtonClick = (index:number) => {
-    //to be implemented
-  };
+  
 
   const renderCards = () => {
-    return cards.map((card, index) => (
-      <Card key={index} onDelete={() => handleDeleteCardButtonClick(index)} />
+    return cards.map(() => (
+      <Card cardsAmount={cardsAmount} setCardsAmount={setCardsAmount} />
```

Comment:
**nitpick:** I don't see how you display saved values in your cards. You are missing props here - `front`, and `back`

Reply:
But how are you gonna display values that come from backend? You should be able to pass values directly to Card component

Reply:
Okay, gotcha.